### PR TITLE
run rustfmt on std::path

### PR DESCRIPTION
**Pull request by tshepang**

### CODE REVIEW

File: src/libstd/path.rs [hunk 1]
```diff
@@ -296,15 +295,21 @@ impl<'a> Prefix<'a> {
         }
         match *self {
             Verbatim(x) => 4 + os_str_len(x),
-            VerbatimUNC(x,y) => 8 + os_str_len(x) +
-                if os_str_len(y) > 0 { 1 + os_str_len(y) }
-                else { 0 },
+            VerbatimUNC(x, y) => 8 + os_str_len(x) +
```
Comment (marcusklaas) [inline src/libstd/path.rs]: Ughh, rustfmt doesn't do inline if-else expressions well. I'd personally refactor this so that it'd use an intermediate value.

Comment (tshepang) [inline src/libstd/path.rs]: I changed that into a block instead... it's more readable, and gets formatted better by rustfmt.

File: src/libstd/path.rs [hunk 2]
```diff
@@ -298,15 +297,23 @@ impl<'a> Prefix<'a> {
...
+                    1 + os_str_len(y)
+                } else {
+                    0
+                }
+            },
             VerbatimDisk(_) => 6,
-            UNC(x,y) => 2 + os_str_len(x) +
-                if os_str_len(y) > 0 { 1 + os_str_len(y) }
-                else { 0 },
+            UNC(x, y) => 2 + os_str_len(x) +
```
Comment (marcusklaas) [inline src/libstd/path.rs]: Could this be changed into a block as well?

Comment (tshepang) [inline src/libstd/path.rs]: Missed it, thanks

**Comment by aturon** [comment]
@tshepang Sorry, I've been away on paternity leave for the past couple of weeks. This looks great to me.

@bors: r+ rollup

**Comment by tshepang** [comment]
no problem, thanks

**Pull request merged**